vochain/state.go

```diff
@@ -658,13 +667,17 @@ func (v *State) RemoveValidator(address []byte) error {
 }
 
 // Validators returns a list of the validators saved on persistent storage
-func (v *State) Validators(isQuery bool) ([]*models.Validator, error) {
-	if !isQuery {
+// TODO: do 'saved on persistent storage' make sense also when committed == false?
```

Comment:
Nope, in this case and following the other methods I think we should rewrite the comment.
I propose to change it to: `Validators returns a list of the chain validators`